plugins/api-docs/src/alpha.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import {
+  createApiExtension,
+  createApiFactory,
+  createNavItemExtension,
+  createPageExtension,
+  createPlugin,
+  createSchemaFromZod,
+} from '@backstage/frontend-plugin-api';
+import {
+  compatWrapper,
+  convertLegacyRouteRef,
+} from '@backstage/core-compat-api';
+import { ApiEntity } from '@backstage/catalog-model';
+
+import { defaultDefinitionWidgets } from './components/ApiDefinitionCard';
+import { rootRoute, registerComponentRouteRef } from './routes';
+import { apiDocsConfigRef } from './config';
+import {
+  createEntityCardExtension,
+  createEntityContentExtension,
+} from '@backstage/plugin-catalog-react/alpha';
+import { Grid } from '@material-ui/core';
+import { useApp } from '@backstage/core-plugin-api';
+
+function ApiIcon() {
```

Comment:
Just a note that we clearly need a some abstraction to make this easier. Probably just a `<Icon id='kind:api'/>` tbh, but could also imagine `<Icon id='kind:api' fallback={KindApiSystemIcon}/>` being useful. Just discussion though, nothing to change in this PR.

Reply:
Would like me to create and issue for that?

Reply:
Added it as an "Qs for Polish" in #19545, but issue/board item could be good too 👍

Reply:
Added to the board 🙂